scripts/peer/peer.py

```diff
@@ -0,0 +1,273 @@
+"""
+Test run this with:
+
+    cd <dir where peer.py file is>
```

Comment:
Can probably update this now that the path has been decided:

```suggestion
    cd scripts/peer
```